pkg/audit/manager.go

```diff
@@ -1182,7 +1182,7 @@ func violationMsg(constraint *unstructured.Unstructured, enforcementAction util.
 
 func logViolation(l logr.Logger,
 	constraint *unstructured.Unstructured,
-	enforcementAction util.EnforcementAction, resourceGroupVersionKind schema.GroupVersionKind, rnamespace, rname, message string, details interface{}, rlabels map[string]string,
+	enforcementAction []string, resourceGroupVersionKind schema.GroupVersionKind, rnamespace, rname, message string, details interface{}, rlabels map[string]string,
```

Comment:
This is a backwards compatibility concern because it affects semantic logging. We could just log one message per enforcement action, though. Or add a new field (though what value would `enforcementAction` have?)

Reply:
We can probably serialize the array. Something like `deny/warn` in-case if enforcemet point is duplicated in `scopedEA` config.

Reply:
to ensure backwards compat, the existing enforcementAction should remain untouched. lets create a new field instead.